vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/Serializer.java

```diff
@@ -42,15 +42,31 @@
  *
  */
 public final class Serializer {
-
+	/**
+	 * Constructor.
+	 */
 	private Serializer() {
 
 	}
 
+	/**
+	 * Creates a String representation of a given {@link Rule}. Example: "p(?X) :-
```

Comment:
I would explain what the example represents, perhaps link to getString(Literal literal), and would include existential variables in our example. We should also link to our syntax wiki.